ddtrace/internal/sma.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import division
+
+
+# The window size should be chosen so that the look-back period is
+# greater-equal to the agent API's timeout. Although most tracers have a
+# 2s timeout, the java tracer has a 10s timeout, so we set the window size
```

Comment:
Just to make sure: the window has to be the same across all trace clients?

Reply:
No, it doesn't have to be. We went with that initially but any tracer can change.